File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/good/EntryPriceVat.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.good;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryPriceVat {

    @SerializedName("price_raw")
    @Expose
    private Integer priceRaw;
    @SerializedName("price")
    @Expose
    private String price;

    public Integer getPriceRaw() {
        return priceRaw;
    }

    public void setPriceRaw(Integer priceRaw) {
        this.priceRaw = priceRaw;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/good/EntryPriceVat.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.good

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryPriceVat {

    @SerializedName("price_raw")
    @Expose
    var priceRaw: Int? = null
    @SerializedName("price")
    @Expose
    var price: String? = null

}
